docs-src/content/_index.md

```diff
@@ -2,11 +2,239 @@
+          {
+            "word": "<eps>",
+            "confidence": 0.108
+          }
+        ],
+        "startTime": "2.120s"
+      },
+      {
+        "duration": "0.010s",
+        "arcs": [
+          {
+            "word": "<eps>",
+            "confidence": 1.0
+          }
+        ],
+        "startTime": "2.288s"
+      },
+      {
+        "duration": "0.093s",
+        "arcs": [
+          {
+            "word": "A",
+            "confidence": 0.620
+          },
+          {
+            "word": "<eps>",
+            "confidence": 0.233
+          },
+          {
+            "word": "ISN'T",
+            "confidence": 0.108
+          },
+          {
+            "word": "THE",
+            "confidence": 0.039
+          }
+        ],
+        "startTime": "2.298s"
+      },
+      {
+        "duration": "0.005s",
+        "arcs": [
+          {
+            "word": "<eps>",
+            "confidence": 1.0
+          }
+        ],
+        "startTime": "2.391s"
+      },
+      {
+        "duration": "0.273s",
+        "arcs": [
+          {
+            "word": "NEW",
+            "confidence": 0.661
+          },
+          {
+            "word": "<UNK>",
+            "confidence": 0.129
+          },
+          {
+            "word": "YOUR",
+            "confidence": 0.107
+          },
+          {
+            "word": "YOU",
+            "confidence": 0.102
+          }
+        ],
+        "startTime": "2.396s"
+      },
+      {
+        "duration": "0s",
+        "arcs": [
+          {
+            "word": "<eps>",
+            "confidence": 1.0
+          }
+        ],
+        "startTime": "2.670s"
+      },
+      {
+        "duration": "0.420s",
+        "arcs": [
+          {
+            "word": "DAY",
+            "confidence": 0.954
+          },
+          {
+            "word": "TODAY",
+            "confidence": 0.044
+          },
+          {
+            "word": "<UNK>",
+            "confidence": 0.002
+          }
+        ],
+        "startTime": "2.670s"
+      },
+      {
+        "duration": "0.270s",
+        "arcs": [
+          {
+            "word": "<eps>",
+            "confidence": 1.0
+          }
+        ],
+        "startTime": "3.090s"
+      }
+    ]
+  }
+}
+```
+</details>
+
+
+## Formatted output
+
+Speech recognition systems typically output the words that were spoken, with no formatting. For example, utterances with numbers in might return “twenty seven bridges”, and “the year two thousand and three”. Cubic has the option to enable basic formatting of speech recognition results:
+
+* Numbers: “cobalt’s atomic number is twenty seven” -> “Cobalt’s atomic number is 27”
+* Truecasing: “the iphone was launched in two thousand and seven” -> “The iPhone was launched in 2007”
+* Ordinals: “summer solstice is twenty first june” -> “Summer solstice is 21st June”
+
+Note that word level timestamps and confidences aren’t supported when formatting is enabled.
+
+
+## Obtaining Cubic
```

Comment:
In the final paragraph, we only mention the go sdk; we are probably ready to mention the python sdk as well.  (I think we do at the top, so this is currently inconsistent.)

Reply:
Done